From 383ef4bc814da5a65c69b34439dc4addfdb47822 Mon Sep 17 00:00:00 2001
From: mytja <mitja@severkar.eu>
Date: Sun, 22 Dec 2024 01:22:45 +0100
Subject: [PATCH] Properly apply "Add properties to disable some
 features/commands/states that the ble chip declared but doesnt actually
 support"

Skill issue by my side.
---
 system/gd/hci/controller.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/system/gd/hci/controller.cc b/system/gd/hci/controller.cc
index b7627e4..2bfb80f 100644
--- a/system/gd/hci/controller.cc
+++ b/system/gd/hci/controller.cc
@@ -19,6 +19,7 @@
 #include <android-base/strings.h>
 #include <android_bluetooth_flags.h>
 
+#include <base/strings/string_split.h>
 #include <future>
 #include <memory>
 #include <string>
-- 
2.34.1

